[2024-05-28T15:47:31.868+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-28T15:47:31.888+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stored_procedures.bash_task_end manual__2024-05-28T15:47:11.852982+00:00 [queued]>
[2024-05-28T15:47:31.897+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stored_procedures.bash_task_end manual__2024-05-28T15:47:11.852982+00:00 [queued]>
[2024-05-28T15:47:31.898+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-28T15:47:31.910+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): bash_task_end> on 2024-05-28 15:47:11.852982+00:00
[2024-05-28T15:47:31.918+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1742) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-28T15:47:31.919+0000] {standard_task_runner.py:63} INFO - Started process 1744 to run task
[2024-05-28T15:47:31.920+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'stored_procedures', 'bash_task_end', 'manual__2024-05-28T15:47:11.852982+00:00', '--job-id', '165', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmp4dl93e3q']
[2024-05-28T15:47:31.923+0000] {standard_task_runner.py:91} INFO - Job 165: Subtask bash_task_end
[2024-05-28T15:47:31.973+0000] {task_command.py:426} INFO - Running <TaskInstance: stored_procedures.bash_task_end manual__2024-05-28T15:47:11.852982+00:00 [running]> on host 29c09f03cc50
[2024-05-28T15:47:32.055+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Yamin' AIRFLOW_CTX_DAG_ID='stored_procedures' AIRFLOW_CTX_TASK_ID='bash_task_end' AIRFLOW_CTX_EXECUTION_DATE='2024-05-28T15:47:11.852982+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-28T15:47:11.852982+00:00'
[2024-05-28T15:47:32.056+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-28T15:47:32.068+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-28T15:47:32.070+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Here is your bash message at the end of your DAG"']
[2024-05-28T15:47:32.083+0000] {subprocess.py:86} INFO - Output:
[2024-05-28T15:47:32.085+0000] {subprocess.py:93} INFO - Here is your bash message at the end of your DAG
[2024-05-28T15:47:32.086+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-28T15:47:32.087+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-28T15:47:32.117+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=stored_procedures, task_id=bash_task_end, execution_date=20240528T154711, start_date=20240528T154731, end_date=20240528T154732
[2024-05-28T15:47:32.175+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-28T15:47:32.201+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-28T15:47:32.205+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
